source/dem/force_chains_visualization.cc

```diff
@@ -294,6 +293,201 @@ ParticlesForceChains<dim, contact_model, rolling_friction_model>::
             }
         }
     }
+  // Doing the same calculations for local-ghost particle pairs
+
+  // Looping over ghost_adjacent_particles with iterator
+  // adjacent_particles_iterator
```

Comment:
This comment is not very useful because the next line is exactly that. Maybe something like loop over ghost_adjacent particle pairs.